src/main/scala/org/broadinstitute/hail/variant/Genotype.scala

```diff
@@ -194,15 +204,18 @@ class Genotype(private val _gt: Int,
     b.result()
   }
 
-  def pAB(theta: Double = 0.5): Option[Double] = ad.map { case Array(refDepth, altDepth) =>
-    val d = new BinomialDistribution(refDepth + altDepth, theta)
-    val minDepth = refDepth.min(altDepth)
-    val minp = d.probability(minDepth)
-    val mincp = d.cumulativeProbability(minDepth)
-    (2 * mincp - minp).min(1.0).max(0.0)
+  def pAB(theta: Double = 0.5): Option[Double] = ad.map {
+    case Array(refDepth, altDepth) =>
```

Comment:
Any reason in particular you changed the indentation here?  For anonymous pattern matching functions with one alternative, I prefer the case just after the open curly brace.